### PR TITLE
[Feature-8030][docs] Add sqoop task doc

### PR DESCRIPTION
**Pull request by baihongbin**
【WeOpen-Star】

## Purpose of the pull request

:gift: Add sqoop task docs 

## Brief change log

:ledger: Added sqoop related documents in the docs folder

## Verify this pull request

:bell: This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: docs/docs/en/guide/task/sqoop.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Sqoop Node
+
+## Overview
+
+Sqoop task type for executing Sqoop application. The workers run `sqoop` to execute  sqoop tasks.
```
Comment (EricGao888) [inline docs/docs/en/guide/task/sqoop.md:5]:
```suggestion
Sqoop task type for executing Sqoop application. The workers run `sqoop` to execute sqoop tasks.
```

File: docs/docs/zh/guide/task/sqoop.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+![sqoop_task02](../../../../img/tasks/demo/sqoop_task02.png)
+
+本样例中的关键配置如下表。
+
+| **任务参数**         | **参数值**                                  |
+|------------------|------------------------------------------|
+| 任务名称             | sqoop_mysql_to_hive_test                 |
+| 流向               | import                                   |
+| 数据来源 - 类型        | MYSQL                                    |
+| 数据来源 - 数据源       | MYSQL MyTestMySQL（MyTestMySQL是我的测试数据源名称） |
```
Comment (EricGao888) [inline docs/docs/zh/guide/task/sqoop.md]:
```suggestion
| 数据来源 - 数据源       | MYSQL MyTestMySQL（您可以将MyTestMySQL改成您自己取的数据源名称） |
```

File: docs/docs/en/guide/task/sqoop.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+![sqoop_task02](../../../../img/tasks/demo/sqoop_task02.png)
+
+The key configuration in this sample is shown in the following table.
+
+| **Parameter**                       | **Value**                                                         |
+|-------------------------------------|-------------------------------------------------------------------|
+| Job Name                            | sqoop_mysql_to_hive_test                                          |
+| Data Source - Type                  | MYSQL                                                             |
+| Data Source - Datasource            | MYSQL MyTestMySQL(MyTestMySQL is the name of my test data source) |
```
Comment (EricGao888) [inline docs/docs/en/guide/task/sqoop.md]:
```suggestion
| Data Source - Datasource            | MYSQL MyTestMySQL(You could change MyTestMySQL to the name you like) |
```

**Comment by baihongbin** [comment]
@EricGao888  What else should I do for this PR? 😀

**Comment by EricGao888** [comment]
> @EricGao888  What else should I do for this PR? 😀

No, just wait for CI to pass. thx

**Review by EricGao888 — DISMISSED** [review]

**Comment by baihongbin** [comment]
> > @EricGao888  What else should I do for this PR? 😀
> 
> No, just wait for CI to pass. thx

Seems to have failed

**Comment by EricGao888** [comment]
> > > @EricGao888  What else should I do for this PR? 😀
> 
> > 
> 
> > No, just wait for CI to pass. thx
> 
> 
> 
> Seems to have failed

u need to run mvn spotless:apply to fix the formatting errors

**Comment by baihongbin** [comment]
> > > > @EricGao888  What else should I do for this PR? 😀
> > 
> > 
> > > 
> > 
> > 
> > > No, just wait for CI to pass. thx
> > 
> > 
> > Seems to have failed
> 
> u need to run mvn spotless:apply to fix the formatting errors

Hmm, I see. I have fixed the formatting errors.

**Review by davidzollo — APPROVED** [review]
+1
good job, thanks for your first contribution
LGTM

**Pull request merged**